drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c

```diff
@@ -371,6 +371,116 @@ void wifi_nrf_event_proc_cookie_rsp(void *vif_ctx,
 #endif /* CONFIG_NRF700X_STA_MODE */
 #endif /* !CONFIG_NRF700X_RADIO_TEST */
 
+
+
+void configure_tx_pwr_settings(struct nrf_wifi_tx_pwr_ctrl_params *tx_pwr_ctrl_params,
+				struct nrf_wifi_tx_pwr_ceil_params *tx_pwr_ceil_params)
+{
+	tx_pwr_ctrl_params->ant_gain_2g = CONFIG_NRF700X_ANT_GAIN_2G;
```

Comment:
you can use a short variable names as the function name already suggests tx_pwr, this should help with alignment and hence readability. E.g., 

`tx_pwr_ctrl_params` -> `ctrl`
`tx_pwr_ceil_params` -> `ceil`